types/greek-utils/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for greek-utils 1.2
+// Project: https://github.com/vbarzokas/greek-utils
+// Definitions by: Dimitris Kirtsios <https://github.com/dimkirt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version 3.9
```

Comment:
this could be removed I think, unless I've missed something, there is nothing so TS version specific
requiring the current version of TS, this would limit usage of the package
```suggestion
```

Reply:
You are right! I thought I had to define a version for the linter to pass, but yeah no real need for 3.9 or greater